openedx/core/djangoapps/user_api/views.py

```diff
@@ -225,17 +227,14 @@ def get(self, request):
 
         return HttpResponse(form_desc.to_json(), content_type="application/json")
 
-    @method_decorator(require_post_params(DEFAULT_FIELDS))
-    @method_decorator(csrf_protect)
+    @method_decorator(csrf_exempt)
```

Comment:
I may be missing some context, but why is the post request to create a new user CSRF exempt?